apps/web/src/components/app/player/PlayerTrackInfo.test.tsx

```
@@ -0,0 +1,119 @@
+
+vi.mock('@/components/ui/slider', () => ({
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  Slider: ({ value, onChange, onMouseEnter, onMouseLeave, onPointerDown, onPointerUp }: any) => (
+    <div
+      data-testid="mock-slider"
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+      onPointerDown={onPointerDown}
+      onPointerUp={onPointerUp}
+      onClick={() => onChange(50)}
+    >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)